### DISCUSSION TITLE
About the variables in the scoring formula.

### DISCUSSION THREAD

**Discussion by billcheung-git** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#78] [Q&A]
Hello guys,
our team is a little bit confused by the scoring formula provided in the Metrics. We do not understand the variables in the formula, so please tell us if you know the meaning of the variables, that would help us a lot. Thank you!

![image](https://user-images.githubusercontent.com/85786599/121772551-8d453780-cba8-11eb-86e2-7d9d1ee44290.png)

**Comment by EY-glowglowglow** [comment]
Hi there,

I'd suggest you check out this article on F1 scores - https://en.wikipedia.org/wiki/F-score
